CHANGELOG.md

```diff
@@ -0,0 +1,50 @@
+# Changelog
```

Comment:
Although this might have been originally included in code-gov-web, could we move this to https://github.com/GSA/code-gov-data/tree/master/schemas instead?  The schema is used on both the front and back end, so I think it makes more sense in code-gov-data, no?  @froi , thoughts?

Reply:
I agree. We should keep all schema related changes in that repo.